backup.sh

```diff
@@ -74,23 +81,50 @@ if [ $? -ne 0 ]; then
 fi
 
 # The $BKPMSG variable will print the error message, you can use it if you're planning on sending an email
-BKPMSG=$(curl -s --cookie $COOKIE_FILE_LOCATION --header "X-Atlassian-Token: no-check" -H "X-Requested-With: XMLHttpRequest" -H "Content-Type: application/json"  -X POST $RUNBACKUP_URL -d "{\"cbAttachments\":\"${ATTACHMENTS}\" }" )
+BKPMSG=$(curl -s --cookie $COOKIE_FILE_LOCATION $RUNBACKUP_URL \
+    -X POST \
+    -H 'DNT: 1' \
+    -H 'Content-Type: application/json' \
+    -H 'Accept: application/json, text/javascript, */*; q=0.01' \
+    -H 'X-Requested-With: XMLHttpRequest' \
+    --data-binary "{\"cbAttachments\":\"${ATTACHMENTS}\", \"exportToCloud\":\"true\" }" )
 
 # Checks if we were authorized to create a new backup
-if [ "$(echo "$BKPMSG" | grep -c Unauthorized)" -ne 0 ]  || [ "$(echo "$BKPMSG" | grep -ic "<status-code>401</status-code>")" -ne 0 ]; then
-    echo "ERROR: authorization failure"
-    exit
+if [ $FILEPREFIX = "JIRA" ]; then
```

Comment:
In this section there should be something like this, to abort when to many tries:
```
if [ "$(echo "$BKPMSG" | grep -ic "Backup frequency is limited")" -ne 0 ]; then
      echo "ERROR: Backup frequency is limited"
      exit
fi
```